types/to-regex-range/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for to-regex-range 5.0
+// Project: https://github.com/micromatch/to-regex-range
+// Definitions by: Richie Bendall <https://github.com/Richienb>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace toRegexRange {
+    interface Options {
+        capture?: boolean;
+        shorthand?: boolean;
+        relaxZeros?: boolean;
+    }
+}
+
+declare function toRegexRange(
+    min: number | string,
+    max?: number | string,
```

Comment:
should the `max` be either undefined, or when present, a `number`?
https://github.com/micromatch/to-regex-range/blob/master/index.js#L21

Reply:
Stringified numbers are also allowed: https://github.com/jonschlinkert/is-number/blob/master/index.js#L14-L16

Reply:
oh, cool, you're right, I've done only first part :)